ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/loopback_interfaces.schema.yml

```diff
@@ -0,0 +1,72 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  loopback_interfaces:
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+          display_name: Loopback Interface
+        description:
+          type: str
+          description: description
+        shutdown:
+          type: bool
+        vrf:
+          type: str
+          display_name: VRF
+        ip_address:
+          type: str
+          display_name: IPv4 Address
+        ip_address_secondaries:
+          type: list
+          items:
+            type: str
+            display_name: IPv4 Address
+        ipv6_enable:
+          type: bool
+        ipv6_address:
+          type: str
+          display_name: IPv6 Address
+        ip_proxy_arp:
+          type: bool
+        ospf_area:
+          type: str
+          display_name: OSPF Area
+        mpls:
+          type: dict
+          keys:
+            ip:
+              type: bool
+            ldp:
+              type: dict
+              keys:
+                interface:
+                  type: bool
+        eos_cli:
+          type: str
+          display_name: Multiline eos cli
+        isis_enable:
+          type: str
+          display_name: ISIS Instance
+        isis_passive:
+          type: bool
+        isis_metric:
+          type: int
```

Comment:
Should we have a convert_types: str on these int variables? Same thing for node segment indices as well.

Reply:
Hi, in all files **isis_metric** is having only integer value and no string value. 
and for node_segment i didn't understand what must be the primary_key.
So i thought use of **convert_types** can be ignored. let me know if i should add convert_types.

Reply:
I am not 100% sure but i believe convert_types can be used if the target variable should be int, but it can be input as a string. So for example if you input isis_metric: "50", that is a string that can be converted to an int. Maybe @ClausHolbechArista or @carlbuchmann  can shed some light.

Reply:
Correct. For most (all?) integer values we should add `convert_types: ['str']` and if we have text inputs which _could_ be pure numbers, we should do the reverse as well.